src/App.js

```diff
@@ -63,12 +62,12 @@ function App({ defaultSettings = initial }) {
                 <About />
               </Route>
 
-              <ConnectedRoute path="/" settings={settings}>
+              <ConnectedRoute path="/" exact settings={settings}>
                 <Studio settings={settings} />
               </ConnectedRoute>
 
-              <Route path="*">
-                <NotFound />
+              <Route path="/*">
+                <Redirect to="/" />
```

Comment:
Just wondering (didn't test this): Will this work with sub-paths?

Reply:
Yes. I just also added `exact` to the `settings` and `about` route, so `/settings/dfsdgsg` is also redirected to `/`.